package.json

```diff
@@ -88,5 +88,8 @@
   "browserslist": [
     ">0.25%",
     "not dead"
-  ]
+  ],
+  "resolutions": {
+    "probe-image-size": "^7.0.0"
```

Comment:
That's the solution proposed from those linked issues. The error was occuring due to an old version of that library but the API of the package hasn't changed so it can be updated to this version and it still works fine without showing the error